Sources/Workspace/ManagedDependency.swift

```diff
@@ -1,54 +1,53 @@
 /*
  This source file is part of the Swift.org open source project
-
+ 
```

Comment:
nit: whitespace-only lines in this file

Reply:
also in `PinsStore.swift`

Reply:
yea not sure what happened here, usually Xcode takes care of those